src/fidesctl/api/ctl/ui.py

```diff
@@ -0,0 +1,31 @@
+"""Various functions to help serve the UI from our server"""
```

Comment:
I tried to architect this file such that we can reuse some of its logic in fidesctl+. The idea would be fidesctl+ would also be able to use the `get_path_to_ui_file` for its own UI files.